src/views/Projects/index.jsx

```diff
@@ -50,22 +73,32 @@ export default class Projects extends Component {
     return (
       <Fragment>
         <header className={classes.header}>
-          <Typography variant="display2" align="center">
+          <Typography variant="display2" align="center" color="primary">
             Codetribute
           </Typography>
-          <Typography variant="subheading" align="center">
-            Find your first code contribution with Mozilla
-          </Typography>
-        </header>
-        <main className={classes.container}>
-          <TextField
-            label="Search"
-            fullWidth
+          <div className={classes.flexContainer}>
+            <div className={classes.flex} />
+            <Typography
+              className={classes.highlightedText}
+              variant="subheading"
+              align="center">
+              Find your first code contribution with Mozilla
+            </Typography>
+            <div className={classes.flex} />
+          </div>
+          <SearchBox
             value={this.state.searchTerm}
-            className={classes.search}
             onChange={this.handleTextInputChange}
           />
-          <Grid container spacing={16}>
+        </header>
+        <main className={classes.container}>
+          <Typography
+            variant="headline"
+            align="center"
+            className={classes.projects}>
+            Projects
+          </Typography>
```

Comment:
We can save space by removing this Typography component. The search box already says "Search Project". Let's give it a try.